src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_correlation_property : SagaPersisterTests<SagaWithoutCorrelationProperty, SagaWithoutCorrelationPropertyData>
```

Comment:
The key here is that there is a custom finder for the start message, should we reflect this in the name?

When_completing_a_saga_started_by_custom_finder perhaps?